[2022-02-03 18:49:58,107] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [queued]>
[2022-02-03 18:49:58,114] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [queued]>
[2022-02-03 18:49:58,115] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 18:49:58,115] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-02-03 18:49:58,115] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 18:49:58,122] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-29T09:00:00+00:00
[2022-02-03 18:49:58,134] {standard_task_runner.py:54} INFO - Started process 9466 to run task
[2022-02-03 18:49:58,205] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-29T09:00:00+00:00', '--job_id', '9', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmptbn00ohj']
[2022-02-03 18:49:58,206] {standard_task_runner.py:78} INFO - Job 9: Subtask twitter_aluraonline
[2022-02-03 18:49:58,251] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-29T09:00:00+00:00 [running]> on host SPVALN1-612R9D4.pitagoras.apollo.br
[2022-02-03 18:49:58,321] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-02-03 18:49:58,323] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z
[2022-02-03 18:50:02,744] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 18:50:03,021] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe482by2mxnbz8dpqw7pxjisa7725
[2022-02-03 18:50:03,022] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 18:50:03,353] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe482bx5z0ybbfbfygogyu2g66f7h
[2022-02-03 18:50:03,354] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 18:50:03,632] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-29T09:00:00.00Z&end_time=2022-01-30T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe4829tcg2mvif567x5j88bmezywt
[2022-02-03 18:50:03,633] {logging_mixin.py:112} WARNING - /mnt/d/Alura/engenharia-dados-apache-airflow/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-02-03 18:50:03,884] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220129T090000, start_date=20220203T214958, end_date=20220203T215003
[2022-02-03 18:50:08,093] {local_task_job.py:102} INFO - Task exited with return code 0
